File: Entregavel1/Java/src/NumeroPrimo.java
public class NumeroPrimo{
    public int numero;
    public boolean ePrimo;

    public NumeroPrimo(int n){
        this.numero = n;
        this.ePrimo = true;
        if(n <= 1){
            this.ePrimo = false;
        }else{
            for(int i = 2; i <= Math.sqrt(n); i++){
                if(n % i == 0){
                    this.ePrimo = false;
                    break;
                }
            }
        }
    }
}

File: Entregavel1/JavaScript/NumeroPrimo/numeroPrimo.js
let a = prompt("Digite um número: ");
let ehPrimo = true;

if (a <= 1) {
    console.log("O número " + a + " não é primo");
} else {
    for (let i = 2; i <= Math.sqrt(a); i++) {
        if (a % i == 0) {
            console.log("O número " + a + " não é primo");
            ehPrimo = false;
            break;
        }
    }

    if (ehPrimo) {
        console.log("O número " + a + " é primo");
    }
}